### PR TITLE
Widget Module - checkbox replacement

### PR DESCRIPTION
**Pull request by ctanveer**
#### Description

Widgets checkbox replacement

#### Link to ticket

Please add a link to the GitHub issue being addressed by this change.

### Type

[] Fix
[x] Feature
[] Cleanup 

#### Screenshot of the result

If your change affects the user interface, you should include a screenshot of the result with the pull request.

### CODE REVIEW

File: .dev/docker-compose.yaml [hunk 1]
```diff
@@ -13,18 +13,18 @@ services:
   leantime-dev:
     build: .
     ports:
-      - "8001:8080" #Using ports that are hopefully available. format is: 80XX
+      - "8888:8080" #Using ports that are hopefully available. format is: 80XX
```
Comment (marcelfolaron) [inline .dev/docker-compose.yaml:16]: Can you revert the changes to this file?

**Review by marcelfolaron — CHANGES_REQUESTED** [review]
Looks good. Can you just revert the docker compose file changes?

File: .dev/docker-compose.yaml (hunk 1 above)
Comment (ctanveer) [reply .dev/docker-compose.yaml]: done
